### PR TITLE
bump facia client version

### PR DESCRIPTION
**Pull request by gidsg**
## What does this change?
Update FAPI client

## What is the value of this and can you measure success?
See https://github.com/guardian/facia-scala-client/blob/master/CHANGES.md#206

## Does this affect other platforms - Amp, Apps, etc?
No

CC @guardian/video-developers @janua @Reettaphant

### CODE REVIEW

**Review by akash1810 — APPROVED** [review]

**Comment by Sarah-Jaina** [comment]
I'm not involved with this, please remove me from your mailing list.

**Comment by janua** [comment]
@gidsg Why has all the cached data changed?

File: article/test/ArticleAmpValidityTest.scala [hunk 1]
```diff
@@ -16,7 +16,6 @@ import org.scalatest.DoNotDiscover
     "/uk-news/2016/aug/09/southern-rail-strike-war-of-words-heats-up-on-second-day", // Story package / tone news
     "/football/2016/jul/10/france-portugal-euro-2016-match-report", // Match summary
     "/us-news/live/2016/aug/12/donald-trump-republicans-hillary-clinton-us-election-live", // Live blog
-    "/sport/live/2016/aug/20/rio-2016-olympics-day-15-mo-farah-relays-tom-daley-nicola-adams-football-live", // Sport live blog
-    "/somerset-county-council-partner-zone/2016/sep/21/somerset-offers-opportunities-develop-social-work-career" // Paid for by page
```
Comment (janua) [inline article/test/ArticleAmpValidityTest.scala]: Why was this removed?

**Comment by gidsg** [comment]
@janua I am not 100% sure but I think as bumping the capi model version means that there are some new atom fields which are now coming through in the response.

**Comment by janua** [comment]
@gidsg It would be good to actually look at the diff of one of the files to be 100% sure.

**Comment by janua** [comment]
:+1:
